### DISCUSSION TITLE
use this inside a tkinter GUI frame is it possible

### DISCUSSION THREAD

**Discussion by rath-anshuman** [r0x0r/pywebview#1141] [Ideas]
I am trying from more than 10 days to add this pywebview type GUI inside tkinter frame. I found out that pywebview module is the most stable one as per my views. If it works inside a tkinter frame it would be great

**Comment by r0x0r** [comment]
No, this is not implemented.
This is a popular feature request though. I am not sure if this is possible, so if you feel adventurous, please investigate.

**Reply by rath-anshuman** [reply]
Can you please help me for a way to achieve it? 
Any suggestions!!
🥲

**Reply by r0x0r** [reply]
I have no experience with tkinter. A quick google search pointed me to this https://github.com/Smart-Space/tkwebview2, which is partly based on my code. I have not tested this myself.

**Reply by rath-anshuman** [reply]
Thank you 😁 for your query consideration
